clientapp/app/components/Reusables/ButtonOne.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+interface IButtonOneProps {
+  text: string
+}
+export default function ButtonOne(props:IButtonOneProps) {
+  const {text} = props
+  return (
+    <div>
+      <button className='sm:w-[264px] sm:h-[66px] rounded-2xl' style={{
+        backgroundColor:"#A8BBFF"
```

Comment:
not a fan of this hardcoded background color for the button. we should probably put all colors into a separate folder or file so that our components can import them. this helps us keep a consistent color scheme and it's easier to manage since all of the colors are in one spot.

eg. we could do something like `backgroundColor: colors.buttonPrimary` or even a CSS class called `shared--button-primary` that specially just has the color